tests/func_tests/test_inheritance.py

```diff
@@ -0,0 +1,130 @@
+# coding: utf-8
+import pytest
+from jinja2 import PackageLoader, Template, nodes, Environment
+from jinja2schema.config import Config
+
+from jinja2schema.core import infer
+from jinja2schema.exceptions import MergeException, UnexpectedExpression
+from jinja2schema.model import List, Dictionary, Scalar, Unknown, String, Boolean, Tuple, Number
+from jinja2schema.util import debug_repr
+
+def test_include_1():
+    env = Environment(loader=PackageLoader('tests', 'templates'))
+    struct = infer(env.loader.get_source(env, 'inner_include_1.html')[0], Config(PACKAGE_NAME='tests'))
+    expected_struct = Dictionary({
+        'var': Dictionary(
+            {'x': Scalar(label='x', linenos=[1]), 'y': Scalar(label='y', linenos=[1])},
+            label='var',
+            linenos=[1]
+        ),
+        'more':  Scalar(label='more', linenos=[2]),
+    })
+    assert struct == expected_struct
+
+def test_include_override_1():
```

Comment:
It think at least this test case is wrong. If we render ``inner_include_override_1.html`` with an empty context, both parent's and child's blocks are rendered and the output is the following:
```
<p>I am the default, my name is </p>
<p>Hello, </p>
```
Which means that both ``name`` and ``default_name`` must be present in the expected structure.

As far as I know, Jinja2 does not allow to override blocks of included templates.